src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java

```diff
@@ -214,11 +214,20 @@ public void testDescribeSObjects() throws Exception {
 
         int numDescribes = 0;
         for (String objectType : client.getDescribeGlobalResults().keySet()){
-            DescribeSObjectResult describeResult = client.describeSObject(objectType);
-            numDescribes++;
-            assertNotNull(describeResult);
-            assertEquals(objectType, describeResult.getName());
-            assertEquals(numDescribes, client.getEntityDescribeMap().size());
+        	try {
+        		DescribeSObjectResult describeResult = client.describeSObject(objectType);
+        		numDescribes++;
+                assertNotNull(describeResult);
```

Comment:
Minor, indentation.

Reply:
good catch. I need to configure my IDE to treat a tab as 4 space characters.